plugins/MantisCoreFormatting/core/MantisMarkdown.php

```diff
@@ -193,7 +190,8 @@ protected function blockTableContinue( $line, array $block ) {
 	private function __quote( $line, $block, $fn ) {
 
 		if( $block = call_user_func( 'parent::' . $fn, $line, $block ) ) {
-			$block['element']['attributes']['style'] = $this->inline_style;
+			# set mantis style
+			$block['element']['attributes']['style'] = 'border-color:#847d7d;font-size:13px;';
```

Comment:
Isn't there a better way than hardcoding the size?
It will not look that good if the size of the original text has been changed (tweaked Mantis CSS)